robots/index.markdown

```diff
@@ -1600,6 +1600,39 @@ title: Robots
 				</div>
 			</div>
 		</div>
+                <div class="col-xs-12 col-sm-6 col-lg-3">
+                        <div class="robots-single">
+                                <div class="robots-image-wrapper" style="background-image: url('/assets/images/robots/xArm 5.png');">
```

Comment:
These three images look exactly the same - is there any difference?

Reply:
xArm5 and 6 do look the same, xArm 7 has some more visible differences. These were the files Rob sent me, which were forwarded from an e-mail sent by a UFactory representative.